app/brave_settings_strings.grdp

```diff
@@ -471,23 +471,21 @@
     Help improve Brave's product stability by automatically sending diagnostic reports when the Brave Browser crashes or freezes.
   </message>
   <!-- Request OTR -->
-  <if expr="enable_request_otr">
-    <message name="IDS_SETTINGS_REQUEST_OTR_LABEL" desc="Request OTR setting label">
-      Go Off-The-Record when visiting sensitive sites
-    </message>
-    <message name="IDS_SETTINGS_REQUEST_OTR_SUB_LABEL" desc="Request OTR setting sub-label">
-      Learn more
-    </message>
-    <message name="IDS_SETTINGS_REQUEST_OTR_DEFAULT" desc="Select value">
-      Ask
-    </message>
-    <message name="IDS_SETTINGS_REQUEST_OTR_ALWAYS" desc="Select value">
-      Always
-    </message>
-    <message name="IDS_SETTINGS_REQUEST_OTR_NEVER" desc="Select value">
-      Never
-    </message>
-  </if>
+  <message name="IDS_SETTINGS_REQUEST_OTR_LABEL" desc="Request OTR setting label">
+    Go Off-The-Record when visiting sensitive sites
```

Comment:
@rmcfadden3 I am not sure what terminology we use for this feature. Should it say `Go Private` instead?

Reply:
My gut says we should avoid phrases like `Go Private` because people will already assume their activity in Brave is private. It might create a moment of confusion.

In this case, maybe something like `View sensitive sites off-the-record (sites won't appear in history)` if that's not too long of a string?

Oh, and if there's room, would be cool to add a `Learn more` to https://brave.com/privacy-updates/26-request-off-the-record/, but I get it if there's not enough pixels for that.

Reply:
Thanks, I didn't realize we already had a publicly released article about the feature. In this case either is fine with me.

Reply:
changed to "View sensitive sites off-the-record" in https://github.com/brave/brave-core/commit/c83ae23838b8132c40d9ce007c338abf52804ae9